examples/cpp/Makefile

```
@@ -46,5 +46,10 @@ elliptic1D: elliptic1D.cpp
 elliptic2D: elliptic2D.cpp
 	$(CXX) $(CXXFLAGS) $(INCPATH) -o elliptic2D elliptic2D.cpp $(LIBS)
 
+RK2: RK2.cpp
+        $(CXX) $(CXXFLAGS) $(INCPATH) -o RK2 RK2.cpp $(LIBS)
```

Hi @Amey-kulkarni19 , thank you so much for your contribution!

You want to be very careful here. It looks like you have used spaces, while we want to use tab here. Otherwise, the `make` command will error with the following error message:

```
Makefile:50: *** missing separator (did you mean TAB instead of 8 spaces?).  Stop.
```

So you want:
```suggestion
	$(CXX) $(CXXFLAGS) $(INCPATH) -o RK2 RK2.cpp $(LIBS)
```

---

Also, you need to modify line 35 above where the `all` rule is defined and you want to add your executable to the list of files:

```
all: transport1D schrodinger1D elliptic1D elliptic2D RK2
```